### PR TITLE
Fix/994 gene pg cancer dist

### PR DESCRIPTION
**Pull request by cy**
~wip~

### CODE REVIEW

File: modules/node_modules/@ncigdc/containers/CancerDistributionTable.js [hunk 1]
```diff
@@ -1,32 +1,72 @@
...
 type TProps = {|
   node: Object,
   aggregations: Object,
 |};
 
-const CancerDistributionTableComponent = ({
+const CancerDistributionTableComponent = compose(
+  lifecycle({
+    componentDidMount(): void {
+      this.props.relay.setVariables({
```
Comment (shanewilson) [inline modules/node_modules/@ncigdc/containers/CancerDistributionTable.js]: do you need this? relay query can't see props?

Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/containers/CancerDistributionTable.js]: I don't think so http://stackoverflow.com/questions/34255662/props-in-relay-preparevariables

Comment (shanewilson) [inline modules/node_modules/@ncigdc/containers/CancerDistributionTable.js]: these ajax containers would probably be easier if they were renderers or at least routes so they can pull in the route params

**Comment by shanewilson** [comment]
how is this coming?

**Comment by cy** [comment]
need to make the same changes in the CancerDistributionChart

**Comment by cy** [comment]
@shanewilson this is ready for review

**Pull request merged**